api/tests/routes/pro/get_offerer_stats_v2_test.py

```diff
@@ -14,147 +15,174 @@
-    )
-
-    collective_offer2 = educational_factories.CollectiveOfferFactory(
-        venue__managingOfferer=user_offerer.offerer, validation=OfferValidationStatus.APPROVED
-    )
-    collective_stock2 = educational_factories.CollectiveStockFactory(collectiveOffer=collective_offer2)
-
-    educational_factories.CollectiveBookingFactory(
-        collectiveStock=collective_stock2, status=CollectiveBookingStatus.CONFIRMED
-    )
-
-    collective_offer3 = educational_factories.CollectiveOfferFactory(
-        venue__managingOfferer=user_offerer.offerer, validation=OfferValidationStatus.APPROVED
-    )
-    collective_stock3 = educational_factories.CollectiveStockFactory(collectiveOffer=collective_offer3)
-    educational_factories.CollectiveBookingFactory(
-        collectiveStock=collective_stock3, status=CollectiveBookingStatus.REIMBURSED
-    )
-
-    for _ in range(3):
-        offers_factories.OfferFactory(
-            venue__managingOfferer=user_offerer.offerer, validation=OfferValidationStatus.PENDING
-        )
-    for _ in range(2):
-        educational_factories.CollectiveOfferFactory(
-            venue__managingOfferer=user_offerer.offerer, validation=OfferValidationStatus.PENDING
+class Return200Test:
+    num_queries = testing.AUTHENTICATION_QUERIES
+    num_queries += 1  # check user_offerer exists
+    num_queries += 1  # select offerer_address
+    num_queries += 1  # count active offers
+    num_queries += 1  # count active collective_offers
+    num_queries += 1  # count pending offers
+    num_queries += 1  # count collective_offers
+
+    def test_get_offerer_stats(self, client):
+        user_offerer = offerers_factories.UserOffererFactory()
+
+        offer = offers_factories.OfferFactory(
+            venue__managingOfferer=user_offerer.offerer, validation=OfferValidationStatus.APPROVED
         )
-    educational_factories.CollectiveOfferTemplateFactory(
-        venue__managingOfferer=user_offerer.offerer, validation=OfferValidationStatus.APPROVED
-    )
-    educational_factories.CollectiveOfferTemplateFactory(
-        venue__managingOfferer=user_offerer.offerer, validation=OfferValidationStatus.PENDING
-    )
-    educational_factories.CollectiveOfferFactory(
-        venue__managingOfferer=user_offerer.offerer, validation=OfferValidationStatus.REJECTED
-    )
-
-    response = client.with_session_auth(user_offerer.user.email).get(f"/offerers/{user_offerer.offerer.id}/v2/stats")
-
-    assert response.status_code == 200
-    assert response.json == {
-        "publishedPublicOffers": 1,
-        "publishedEducationalOffers": 1,
-        "pendingPublicOffers": 3,
-        "pendingEducationalOffers": 3,
-    }
-
-
-def test_get_offerer_stats_with_no_public_offers(client):
-    user_offerer = offerers_factories.UserOffererFactory()
-    for _ in range(2):
-        educational_factories.CollectiveOfferFactory(
+        for _ in range(3):
+            offers_factories.StockFactory(offer=offer)
```

Comment:
```suggestion
        offers_factories.StockFactory.create_batch(size=3, offer=offer)
```
(style) et pour toutes les boucles similaires plus bas, dans un commit de style séparé par exemple ?